frontend/src/views/web/our-team.html

```diff
@@ -1,124 +1,69 @@
 <div class="white-bg view-container">
     <main-header></main-header>
     <div class="grad-container">
-        <h4 class="team-heading">Team Awesome!</h4>
+        <h4 class="team-heading">{{ourTeam.coreTeamType}}</h4>
         <div class="row">
-            <!-- integrate this card with API -->
-            <div class="col s12 l3 ">
+            <!-- This card is for Core Team -->
+            <div class="col s12 l3" ng-repeat="data in ourTeam.coreTeamDetails">
                 <div class="team-card">
                     <div class="team-link-head">
                         <span>
-    				<a href="http://deshraj.github.io/"><i class="fa fa-linkedin"></i></a>
-    			</span>
+                    <a href="{{data.linkedin_url}}"><i class="fa fa-linkedin"></i></a>
```

Comment:
@TroJan If we are using the `URLField` then I think that we don't need client side check because the fields in the database will not be saved until it is a valid URL & if we are saving a valid URL then we have to display it. 
I am suggesting to change it into `CharField` so that if someone doesn't have `github`, `linkedin` or `personal` profile links then he/she will be listed in the Team Page. What are your views abour it?